FrontDesk/src/FrontDesk.Core/Handlers/EmailConfirmationHandler.cs

```diff
@@ -15,7 +15,7 @@ namespace FrontDesk.Core.Handlers
   /// <summary>
   /// This handler responds to incoming messages saying a user has confirmed an appointment
   /// </summary>
-  public class EmailConfirmationHandler : INotificationHandler<AppointmentConfirmLinkClickedIntegrationEvent>
+  public class EmailConfirmationHandler : INotificationHandler<AppointmentConfirmLinkClickedEvent>
```

Comment:
Why are we no longer referring to these as "Integration Events" ?

Reply:
This one was interesting:
Previously, it was an integration event, because we just had the one class and no shared projects.

Currently, we have:
- `VetClinicPublic.Contracts.AppointmentConfirmLinkClickedIntegrationEvent` - fired from VetClinicPublic
- `FrontDesk.Core.Events.IntegrationEvents.AppointmentConfirmLinkClickedEvent` - which is currently just being used to resolve a MediatR publish handler `EmailConfirmationHandler` to do the work

So I see the confusion.
I _could_ get rid of the MediatR handler and just do the work in the `Consumer` of the integration event?

But I should definitely update the names.

Reply:
How about this?
The first one (from VetClinicPublic) is an Integration Event because it spans projects (bounded contexts).
The second one (used by MediatR in FrontDesk) could instead be modified to be a Command (i.e. `ConfirmAppointmentCommand`) which would be created and sent from the `Consumer` of the integration event.

Does that make logical sense to you?